core/src/main/java/org/apache/accumulo/core/metadata/TabletFile.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.accumulo.core.metadata;
+
+import org.apache.hadoop.fs.Path;
+
+public interface TabletFile<T extends TabletFile<T>> {
```

Comment:
The type parameter T is not used, could it be removed?

```suggestion
public interface TabletFile {
```

Reply:
Good point, I just copied that from AbstractTabletFile where it was used for Comparable so I can drop it.